tide_media.install

```diff
@@ -322,3 +322,28 @@ function tide_media_update_8006() {
   }
   drupal_flush_all_caches();
 }
+
+/**
+ * Enable the field_media_site field in form_display of all media bundles.
+ */
+function tide_media_update_8007() {
+  // Default settings.
+  $default_option = [
+    'type' => 'options_buttons',
+    'region' => 'content',
+    'settings' => [],
+    'third_party_settings' => [],
+  ];
+  $bundle_info_service = \Drupal::service('entity_type.bundle.info');
+  $entity_form_display_storage = \Drupal::entityTypeManager()->getStorage('entity_form_display');
+  $media_bundles = $bundle_info_service->getBundleInfo('media');
```

Comment:
See @anthony-malkoun 's comment on dpc-sdp/tide_site#40 wrt naming of this variable.

Reply:
@GROwen, it gets updated.